Sources/SwiftRefactor/ConvertComputedPropertyToStored.swift

```diff
@@ -0,0 +1,55 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#if swift(>=6)
+public import SwiftSyntax
+#else
+import SwiftSyntax
+#endif
+
+public struct ConvertComputedPropertyToStored: SyntaxRefactoringProvider {
+  
+  public static func refactor(syntax: VariableDeclSyntax, in context: ()) -> VariableDeclSyntax? {
+    guard syntax.bindings.count == 1, let binding = syntax.bindings.first,
+          let accessorBlock = binding.accessorBlock, case let .getter(body) = accessorBlock.accessors, !body.isEmpty else {
+      return nil
+    }
+    
+    var initializer: InitializerClauseSyntax?
+    
+    if body.count > 1, let closure = body.with(\.trailingTrivia, .newline).as(ClosureExprSyntax.self) {
+      initializer = InitializerClauseSyntax(
+        value: FunctionCallExprSyntax(
+          leadingTrivia: .space,
+          calledExpression: closure,
+          leftParen: .leftParenToken(),
+          arguments: {[]}(),
+          rightParen: .rightParenToken()
+        )
+      )
```

Comment:
I don’t think this works because `body` is a `CodeBlockItemListSyntax` and converting that to a `ClosureExprSyntax` will never succeed. The test case also seems to fail.